templates/partials/filters/filter-buttons.html

```diff
@@ -1,4 +1,4 @@
-<div class="filter">
+<div>
```

Comment:
`.filter` is a styling class that adds the necessary amount of margin / padding to this element. Can we keep it here? If it's controlling js, we should just add a separate `.js-` class.